src/main/java/com/amazon/jenkins/ec2fleet/EC2FleetCloud.java

```diff
@@ -816,6 +841,25 @@ private void addNewSlave(final AmazonEC2 ec2, final Instance instance, FleetStat
                 TimeUnit.SECONDS.toMillis(getInitOnlineCheckIntervalSec()));
     }
 
+    private int getCurrentSpareInstanceCount(final FleetStateStats currentState, final int countOfInstances) {
+        final int currentSpareInstanceCount = 0;
+        if(minSpareSize > 0) {
```

Comment:
will fetching the `currentSpareInstanceCount` ever be useful at all outside of this new config/feature? Maybe wanting to know how many instances are 'idle' at a given point